2355/1/koans/triangle.rb

```diff
@@ -0,0 +1,63 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+# rubocop:disable Style/IfUnlessModifier
+# rubocop:disable Style/EmptyCaseCondition
+# rubocop:disable Layout/CaseIndentation
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+# This method smells of :reek:UtilityFunction
+# This method smells of :reek:DuplicateMethodCall
+def triangle_type(first_side, second_side, third_side)
+  case
+    when first_side == second_side && second_side == third_side
```

Comment:
Hi. I'm a mentor from Rubizza. Alena asked to help a bit with reviews, so here am I :)

Everything look just awesome. Good work!

Just one complaint about the code style. More precisely, a question. Why did you disable Layout/CaseIndentation for Rubocop?

We usually don't indent `when`s inside of the `case` block.

Care to explain? 🙂

Reply:
So I disabled Layout/CaseIndentation for Rubocop because it was a mistake that prevented me from passing the code of the hound

Reply:
@Aleksei12942 gotcha, then fix this issue, enable that linter and you're 💯% cool.

Reply:
can it wait until hours 9-10? I'm now in a different city far from the computer

Reply:
@Aleksei12942 sure.